.github/workflows/canister-tests.yml

```diff
@@ -267,6 +267,7 @@ jobs:
           mkdir dist
           touch dist/index.html
           touch dist/about.html
+          touch dist/1.js.gz
```

Comment:
will be removed once #1408 is merged

Reply:
can we rename `1.js.gz` to a particular chunk name? would that be interesting?

Reply:
I thought about this, then realized it probably didn't matter at all since (except for here) we'll never actually see the filename anywhere. But I think it's still a good idea, for instance for debugging. I'll change that.